README.md

```diff
@@ -1,14 +1,44 @@
 # Container Linux Config Transpiler
 
-The config transpiler ("ct" for short) is the utility responsible for transforming a Container Linux instance's configuration from the human-friendly, YAML form into an [Ignition][ignition] configuration. While it is possible to write Ignition configs directly, CoreOS recommends that this tool be used instead. More details about the reasoning and the overall design of Ignition and this project are provided in the [documentation][overview].
+The Config Transpiler ("ct" for short) is the utility responsible for transforming a human-friendly Container Linux Config into a JSON file. This resulting file can be provided to a Container Linux machine when it first boots to provision the machine.
 
-Use the [bug tracker][issues] to report bugs.
+Users of [Matchbox][matchbox] don't need to directly use ct, Matchbox will take ct configs directly.
+
+[matchbox]: https://github.com/coreos/matchbox
+
+## Documentation
+
+If you're looking to begin writing configs for your Container Linux machines, check out the [getting started][get-started] documentation.
+
+The [configuration][config] documentation is a comprehensive resource of what options can be in a Container Linux Config.
+
+For a more in depth view on what ct is, why it exists, and how it relates to Ignition, take a look at the [overview][overview] document.
+
+Please use the [bug tracker][issues] to report bugs.
 
 [ignition]: https://github.com/coreos/ignition
 [issues]: https://github.com/coreos/bugs/issues/new?labels=component/ct
 [overview]: doc/overview.md
+[get-started]: doc/getting-started.md
+[config]: doc/configuration.md
+
+## Examples
+
+There are plenty of small, self-contained examples over [in the documentation][examples].
```

Comment:
There are plenty of small, self-contained examples [in the documentation][examples].